src/loader/loader-js-error-event.js

```diff
@@ -5,7 +5,18 @@ loader('cwr', 'abc123', '1.0', 'us-west-2', './rum_javascript_telemetry.js', {
     allowCookies: true,
     dispatchInterval: 0,
     metaDataPluginsToLoad: [],
-    eventPluginsToLoad: [new JsErrorPlugin()],
+    eventPluginsToLoad: [
+        new JsErrorPlugin({
+            shouldIgnoreError: (errorEvent) => {
+                const patterns = [/ResizeObserver loop/];
```

Comment:
Why are we passing this default here?

Shouldn't this be in the plugin itself?

Reply:
Not sure what you mean here - can you clarify a bit more?

Reply:
Where is this file used?

is it a test/fixture file?

if so, should be noted in the filename

Reply:
Loader files are used to load the web client in our test applications - we have different loader files for different test applications, as it helps us isolate the functionalities and behaviors we want to test.

I think the naming convention we have indicates that decently